package.json

```diff
@@ -1,17 +1,17 @@
 {
-    "name": "o-squishy-list",
-    "version": "0.0.0",
-    "description": "UI tabs",
-    "scripts": {
-        "test": "./node_modules/karma/bin/karma start karma.conf.js"
-    },
-    "devDependencies": {
-        "phantomjs": "^1.9.7-3",
-        "browserify": "^3.41.0",
-        "karma": "^0.12.6",
-        "karma-jasmine": "^0.1.5",
-        "karma-phantomjs-launcher": "^0.1.4",
-        "karma-browserify": "^0.2.0",
-        "debowerify": "^0.7.0"
-    }
+  "name": "o-squishy-list",
+  "version": "0.0.0",
```

Comment:
http://origami.ft.com/docs/component-spec/modules/

It says that it shouldn't include it

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.